### ISSUE TITLE
NTR:"developmentally related to" for ecosystems?

### ISSUE THREAD

**Issue by pbuttigieg** [oborel/obo-relations#73]
I know this class requires GO:0032502, but could a similar class be considered for the (natural) development of one ecosystem into another (e.g. ecological succession)?

We could also have ecological succession as an ENVO process and use `has input` and `has output`.

**Comment by cmungall** [comment]
Yes, I think we need another parallel branch but using a different terminology ('development' could mean different things in some ENVO contexts).

Do we have a definition of ecological succession to start with?

**Comment by nlharris** [comment]
This is a very old issue -- is it still relevant and unaddressed?
